libvast/vast/bitmap_algorithms.hpp

```diff
@@ -585,7 +585,7 @@ bool any(const Bitmap& bm) {
     using word_type = typename Bitmap::word_type;
     for (auto b : bit_range(bm)) {
       auto x = b.data();
-      if (b.size() <= word_type::width)
+      if (b.size() < word_type::width)
```

Comment:
Can you briefly explain that the algorithm still computes the right result for `b.size() == word::width`? Or was that a bug?

Reply:
The guarded instruction is a no-op if `b.size() == word_type::with`. In detail: `msb_fill` returns 0 in that case, and `(x |= 0) == x`.